draft-ietf-quic-transport.md

```diff
@@ -3720,7 +3720,7 @@ containing that information is acknowledged.
   unless the endpoint has sent a RESET_STREAM for that stream.  Once an endpoint
   sends a RESET_STREAM frame, no further STREAM frames are needed.
 
-* ACK frames carry the most recent set of acknowledgements and the Ack Delay
+* ACK frames carry the most recent set of acknowledgements and the ACK delay
```

Comment:
This refers to the field so it should be "ACK Delay field"

Reply:
OK

Reply:
Actually, this seems to refer to the value, not the field. ("ACK frames carry" the value.)

Reply:
how about "and the acknowledgement delay from the largest acknowledged packet ({{host-delay}})" ?